WORKSPACE

```diff
@@ -45,8 +49,15 @@ http_archive(
     sha256 = ENVOY_SHA256,
 )
 
-load("@envoy//bazel:repositories.bzl", "GO_VERSION", "envoy_dependencies")
+#local_repository(
+#     name = "envoy",
+#     path = LOCAL_ENVOY_PROJECT
+#)
 
+load("@envoy//bazel:api_repositories.bzl", "envoy_api_dependencies")
+envoy_api_dependencies()
```

Comment:
why did you have to add this?

Reply:
Envoy is splitting /api to external repository. It's WIP and I am seeing error when I introduce envoy using local_repository. 

With this fix, proxy build works perfectly no matter we introduce envoy by http or by local_repository.